core/src/main/java/hudson/model/UpdateCenter.java

```diff
@@ -230,6 +273,74 @@ public InstallationJob getJob(Plugin plugin) {
     }
 
     /**
+     * Get the current connection status.
+     * <p>
+     * Supports a "siteId" request parameter, defaulting to "default" for the default 
+     * update site.
+     * 
+     * @return The current connection status.
+     * @since FIXME
```

Comment:
Curious to know why you'd treat REST-like endpoints differently to the other endpoint methods that get invoked via http calls?